aggsender/rpc/aggsender_rpc.go

```diff
@@ -0,0 +1,68 @@
+package aggsenderrpc
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/cdk-rpc/rpc"
+	"github.com/0xPolygon/cdk/aggsender/types"
+	"github.com/0xPolygon/cdk/log"
+)
+
+const (
+	base10 = 10
+)
+
+type aggsenderStorer interface {
```

Comment:
I would change it to `store`. It is more accurate in my opinion:
 
```suggestion
type aggsenderStore interface {
```

Reply:
The [convention for naming interfaces](https://go.dev/doc/effective_go#interface-names) is adding suffix `-er`, even it's not a valid word in english